CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# 0.1.1
+
+## Fixes
```

Comment:
Hm, do we want to structure it like this? Looking at other packit repos, the changes are always directly under the tag header so I'd maybe follow this for consistency.

Reply:
Oh, right, I was looking at some older changelog entries and didn't realize the style has changed.